### ISSUE TITLE
This library has not been compiled

### ISSUE THREAD

**Issue by prokie** [mongodb-labs/mongo-arrow#82]
With a fresh install of pymongoarrow, running python 3.10.5 I get the following

```
C:\Users\prokie>pip list
Package      Version
------------ -------
numpy        1.22.4
pip          22.1.2
pyarrow      7.0.0
pymongo      4.1.1
pymongoarrow 0.4.0
setuptools   62.4.0

C:\Users\prokie>python
Python 3.10.5 (tags/v3.10.5:f377153, Jun  6 2022, 16:14:13) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import pymongoarrow
C:\Users\prokie\AppData\Roaming\Python\Python310\site-packages\pymongoarrow\__init__.py:33: UserWarning: This library has not been compiled
  warnings.warn("This library has not been compiled")
```
What am I missing?

**Comment by prokie** [comment]
A computer restart solved the issue.

**Comment by ShaneHarvey** [comment]
At first glance I'm not quite sure how that error could happen or why a OS restart would fix it. We should find out how to repro it and update the error to be more descriptive and actionable.

@prokie how did you originally install pymongoarrow? Did you use conda-forge, pip, or compile and install from source?

**Comment by prokie** [comment]
I did "pip install pymongoarrow"

**Comment by blink1073** [comment]
I opened https://jira.mongodb.org/browse/ARROW-104 to track the issue.

**Comment by sabrodigan** [comment]
I have the same issue, clean environment, same versions with the same error message

**Comment by wpritom** [comment]
I'm having the same issue.  OS restart didn't fix it. Anyone found any solution yet?

**Comment by blink1073** [comment]
@wpritom if you can try editing the `pymongoarrow/__init__.py` file locally to remove the `try:` block around `from pymongoarrow.lib import libbson_version` and report the traceback of the `ImportError` it would help us diagnose.

**Comment by blink1073** [comment]
Closing as fixed by #97.  Please re-open an issue if the error persists after the 0.6.0 release (early next week).
